pyproject.toml

```diff
@@ -18,18 +18,19 @@ packages = [
 
 [tool.poetry.dependencies]
 python = ">=3.7.1,<4.0"
-nautobot = "^1.2.0"
+nautobot = "^1.4.0"
```

Comment:
why do we increase this dependency?

Reply:
Probably no reason to really, I can remove.